src/app/views/certificates/certificates.js

```diff
@@ -21,7 +18,7 @@ const statusColors = {
 };
 
 const Certificates = () => {
-  const [isLoading, setIsLoading] = useState(false);
+  const [isLoading] = useState(false);
```

Comment:
what is the point on having useState if you are never going to change its value? You can do just:
```
const isLoading = false;
```

or maybe you don't even need that variable?

Reply:
is being used, i leave the variable